scripts/background.js

```diff
@@ -342,7 +343,16 @@ chrome.runtime.onMessage.addListener(function (message, sender, sendResponse) {
 })
 
 chrome.tabs.onUpdated.addListener(function (tabId, info, tab) {
-  if (info.status === 'complete') {
+  if (info.status === "complete") {
+    // wayback count
+    chrome.storage.sync.get(['wm_count'], function (event) {
+      if (event.wm_count === true) {
+        updateWaybackCountBadge(tab.id, tab.url)
+      } else {
+        updateWaybackCountBadge(tab.id, null)
+      }
+    })
+    // auto save page
     chrome.storage.sync.get(['auto_archive'], function (event) {
```

Comment:
We can merge this and the above `chrome.storage.sync.get(['wm_count'],function(event))`  in a single line as` chrome.storage.sync.get(['wm_count', 'auto_archive'],function(event){}`  and do all the required checks there.

Reply:
Thanks I'll make this change.